### PR TITLE
Add a Progressive Web App manifest

### PR DESCRIPTION
**Pull request by remyleone**
The web app manifest is a simple JSON file that tells the browser about your web application and how it should behave when 'installed' on the user's mobile device or desktop. A typical manifest file includes information about the app name, icons it should use, and the URL that should be opened when the app is launched.

Documentation about adding a web app manifest: https://web.dev/add-manifest/

### CODE REVIEW

File: static/manifest.webmanifest [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "short_name": "Kubernetes",
+  "name": "Kubernetes Documentation",
+  "description": "Kubernetes Documentation website",
```
Comment (sftim) [inline static/manifest.webmanifest]: There's more to the website than documentation, I think.

Comment (sftim) [inline static/manifest.webmanifest]:
```suggestion
  "description": "An open-source system for automating deployment, scaling, and management of containerized applications",
```

File: static/manifest.webmanifest [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "short_name": "Kubernetes",
+  "name": "Kubernetes Documentation",
```
Comment (sftim) [inline static/manifest.webmanifest]: To me, “Kubernetes” works fine as a long and a short name.
```suggestion
  "name": "Kubernetes",
```

**Comment by sftim** [comment]
This looks good to me. I didn't test it at all so far.

**Comment by remyleone** [comment]
@gochist

**Comment by gochist** [comment]
LGTM. Good to learn about web app manifest. I verified that the manifest works well with my Chrome browser. I'm assigning this week's PR wrangler.

/assign @jimangel

File: static/manifest.webmanifest [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "src": "/images/kubernetes-512x512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": "/",
+  "background_color": "#326ce5",
+  "display": "standalone",
```
Comment (sftim) [inline static/manifest.webmanifest]:
```suggestion
  "display": "browser",
```
?

The website isn't made to work like a standalone app.

**Comment by jimangel** [comment]
@remyleone would you please squash your commits?

/lgtm

Thanks!

**Comment by remyleone** [comment]
@jimangel done

**Pull request merged**